lib/jekyll/theme.rb

```diff
@@ -0,0 +1,48 @@
+module Jekyll
+  class Theme
+    attr_reader :name
+
+    def initialize(name)
+      @name = name.downcase.strip
+      raise Jekyll::Errors::MissingDependencyException unless gemspec
```

Comment:
Would you please add a message to this raise as a second argument?

```ruby
unless gemspec
  raise Jekyll::Errors::MissingDependencyException, "#{name} could not be found on your system."
end
```

Also, what motivated your choice to place this logic here? Just an early fail?